components/autofill/Cargo.toml

```
@@ -22,7 +22,7 @@ sync-guid = { path = "../support/guid", features = ["rusqlite_support", "random"
 sync15 = { path = "../sync15", features = ["sync-engine"] }
 thiserror = "1.0"
 types = { path = "../support/types" }
-uniffi = { workspace = true }
+uniffi = { version = "0.23" }
```

nit: we don't need the `version`
```suggestion
uniffi = "0.23"
```